requirements-build.txt

```diff
@@ -20,6 +18,8 @@ wheel==0.36.2
     # via -r requirements-build.in
 
 # The following packages are considered to be unsafe in a requirements file:
+pip==21.3.1
```

Comment:
I'm a bit confused why the build requirements are saying pip 21, and the (runtime?) requirements are pip 23. Is this is mistake, or is there some rationale?

Reply:
Huh, that is indeed weird. From what I could tell CentOS 8 / RHEL 8.4 base images only include python 3.6 which seems to be limited to pip 21 from what I saw. I will double check this though, because you're right, this is inconsistent.